kf_lib_data_ingest/templates/my_study/extract_configs/extract_config.py

```diff
@@ -6,9 +6,9 @@
 information on writing extract config files.
 """
 
-from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common import constants  # noqa F401
```

Comment:
Why do we want some lines not to be checked by the linter? It seem neither exceeding the character limit nor break linting rules?

Reply:
F401 is about having unused imports. Since this file is part of the template ingest package, the unused imports are part of the template for the extract_config.py template.